dashboard/config/blocks/PoemBot/PoemBot_setForegroundEffect.json

```diff
@@ -2,7 +2,7 @@
   "category": "",
   "config": {
     "func": "setForegroundEffect",
-    "blockText": "set foreground effect {EFFECT}",
+    "blockText": "do {EFFECT} effect",
```

Comment:
[this can happen post-playtest] should we rename the func/file for this block?

Reply:
Yeah, that's a good callout. Changing the names of blocks is tough because it requires updating all the levels that use the block, and also breaks existing student projects that use the block. As a result, we've ended up with a bunch of block names that aren't quite right, but aren't worth fixing.
In this case, I held off on doing it as part of this change because I'm not sure whether this block is 100% settled or still iterating. Also, I didn't want to have to update all the playtest levels right away :) I think we should wait until after the playtest and get the block 100% finalized, then change the name before launch.

Reply:
perfect, i think that's the right direction here